lib/sycamore/sycamore/tests/conftest.py

```diff
@@ -13,3 +14,8 @@ def read_local_binary(request) -> Document:
     document.binary_representation = input_stream.readall()
     document.properties["path"] = path
     return document
+
+
+@pytest.fixture(params=[ExecMode.LOCAL, ExecMode.RAY])
+def exec_mode(request):
```

Comment:
I'd add a docstring here so it's clear how this fixture should be used.

Reply:
Ack